library/core/src/ffi/c_str.rs

```diff
@@ -126,35 +126,20 @@ pub struct CStr {
 /// ```
 #[derive(Clone, PartialEq, Eq, Debug)]
```

Comment:
Since all fields are exposed then it's fine to make this `Copy`.

Reply:
thx, added